doc/sphinx/language/core/sections.rst

```diff
@@ -131,6 +131,90 @@ usable outside the section as shown in this :ref:`example <section_local_declara
    Notice the difference between the value of :g:`x'` and :g:`x''` inside section
    :g:`s1` and outside.
 
+.. _visibility-attributes-sections:
+
+Summary of locality attributes in a section
+-------------------------------------------
+
+The following table sums up the locality of vernacular commands in sections, when
+outside the section where it was entered.
+A similar table for :cmd:`Module` can be found
+:ref:`here <visibility-attributes-modules>`.
+
+.. list-table::
+  :header-rows: 1
+
+  * - ``Command``
+    - without attribute
+    - :attr:`local`
+    - :attr:`export`
+    - :attr:`global`
+
+  * - :cmd:`Definition`, :cmd:`Lemma`, :cmd:`Axiom`, ...
+    - same as :attr:`local`
+    - available
+    - attribute not supported
+    - attribute not supported
+
+  * - :cmd:`Notation`
+    - same as :`local`
+    - not available
+    - attribute not supported
+    - attribute not supported
+
+  * - :cmd:`Notation (abbreviation)`
+    - same as :attr:`local`
+    - not available
+    - attribute not supported
+    - attribute not supported
+
+  * - ``Hints`` (and :cmd:`Instance`)
+    - same as :attr:`local`
+    - not available
+    - attribute not supported
+    - attribute not supported
+
+  * - :cmd:`Set` or :cmd:`Unset` a flag
+    - same as :attr:`local`
+    - no effect
+    - in effect
+    - in effect
```

Comment:
How is this different from being "available" or "supported"?

Reply:
Thank you for your comments. This is probably unclear. What I meant was that the setting itself is in effect outside the module or section.

Reply:
I have changed it to "available" for consistency with an explanation of what it means for this table.